Nitric.Sdk/Secret/Secret.cs

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021, Nitric Technologies Pty Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using System;
+using System.Text;
+using Nitric.Proto.Secret.v1;
+using GrpcClient = Nitric.Proto.Secret.v1.SecretService.SecretServiceClient;
+namespace Nitric.Api.Secret
+{
+    public class Secret
+    {
+        public readonly static string LATEST = "latest";
+
+        internal GrpcClient client;
+        public string Name { get; private set; }
```

Comment:
In other classes in this package you use a public readonly field for immutable data. Should you also use this for the Name property?

Reply:
It will have the same effect with the private set modifier, but for consistency I changed it.